### PR TITLE
bleep: 0.0.7 -> 0.0.9

### PR DESCRIPTION
**Pull request by KristianAN**
release-notes: https://github.com/oyvindberg/bleep/releases/tag/v0.0.9

Added a passthru test.

### CODE REVIEW

**Comment by nim65s** [comment]
Result of `nixpkgs-review pr 346909` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bleep</li>
  </ul>
</details>

File: pkgs/by-name/bl/bleep/package.nix [hunk 1]
```diff
@@ -59,6 +60,11 @@ stdenvNoCC.mkDerivation (finalAttrs: {
         --zsh <(bleep install-tab-completions-zsh --stdout) \
     '';
 
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+    command = "bleep --help | grep \"Bleeping\" | awk '{print $5}' | tr -d ')'";
```
Comment (nim65s) [inline pkgs/by-name/bl/bleep/package.nix:65]:
```suggestion
    command = "bleep --help | grep \"Bleeping\" | awk '{print $5}' | tr -d ')'";
    command = "bleep --help | sed -n -e '/Bleeping/s/[^0-9.]//gp'"
```

I guess a single sed call with "show only lines with `Bleeping` and remove all other characters than `0-9.`" is more simple

Comment (nim65s) [reply pkgs/by-name/bl/bleep/package.nix:65]: But also, "[versionCheckHook](https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook) should be preferred"
ref. https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion

Comment (KristianAN) [reply pkgs/by-name/bl/bleep/package.nix:65]: Will this still trigger passthru tests on ofborg? That is the main reason I added it. Without the passthru it hangs for several days on the passthru testing.

**Comment by Gako358** [comment]
What is missing to merge this?

**Pull request merged**